docs/UserGuide.md

```diff
@@ -157,9 +157,9 @@ to configure the list of repositories to analyze and the respective repository l
 Column Name | Explanation
 ----------- | -----------
 Repository's Location | The `GitHub URL` or `Disk Path` to the git repository
-Branch | The branch to analyze in the target repository
-[Optional] File formats<sup>*</sup> | The file formats to analyze in `alphanumerical`. If not provided, the following file formats will be used. `adoc; cs; css; fxml; gradle; html; java; js; json; jsp; md; py; tag; xml`
-[Optional] Ignore Glob List<sup>*</sup> | The list of file path globs to ignore during analysis for each author. More details on the Java glob standard [here](https://javapapers.com/java/glob-with-java-nio/)
+[Optional] Branch | The branch to analyse in the target repository
+[Optional] File formats | The file formats to analyse in `alphanumerical`. If not provided, the following file formats will be used. `adoc, cs, css, fxml, gradle, html, java, js, json, jsp, md, py, tag, xml`
```

Comment:
Why are you making the changes in this line? 

Also, we use American spelling according to our coding convention. `analyse` -> `analyze`.